.github/workflows/log_sizes.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Binary Size Log
+
+on:
+  workflow_dispatch: {}
+#  schedule:
+    # 6am or 7am PT depending on daylight savings
+#    - cron: '0 13 * * *'
+    
+
+jobs:
+  binary-size-log:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: update logs x86
+      uses: docker://ghcr.io/tflm-bot/tflm-ci:0.1
+      with:
+        args: tensorflow/lite/micro/tools/metrics/create_size_log_x86.sh
+    - name: commit logs
+      run: |
+        git config --global user.name 'TFLM-bot'
+        git config --global user.email 'tflm-github-bot@google.com'
+        git commit -am "Automated size log update"
+        git push
```

Comment:
Suggest to comment the "git push" and uncomment the below PR generation. TFLM repo does not allow directly push I believe.